lib/wise_homex/models/ds_energy_location.ex

```diff
@@ -0,0 +1,22 @@
+defmodule WiseHomex.DSEnergyLocation do
+  @moduledoc false
+
+  use WiseHomex.BaseModel
+
+  embedded_schema do
+    belongs_to :ds_energy_group, WiseHomex.DSEnergyGroup
+    belongs_to :household, WiseHomex.Household
+    belongs_to :room, WiseHomex.Room
+
+    field :customer_name, :string
+    field :location_name, :string
+    field :full_location, :string
+    field :full_location_name, :string
+
+    field :parent_full_location, :string
+
+    field :depth, :integer
```

Comment:
Depth is not exposed in the API

Reply:
Good catch. Sorry about that. Now exposed in the API :sunny: